lib/smart_proxy_container_gateway/container_gateway_api.rb

```diff
@@ -1,7 +1,9 @@
 require 'sinatra'
 require 'smart_proxy_container_gateway/container_gateway'
 require 'smart_proxy_container_gateway/container_gateway_main'
+require 'smart_proxy_container_gateway/foreman_api'
 require 'sequel'
+require 'ostruct'
```

Comment:
I'm curious what requires this library.

Reply:
oh it probably dosen't anymore, an earlier version did but i refactored it.  good catch